DEPS

```diff
@@ -606,6 +613,29 @@ hooks = [
       'src/third_party/dart/third_party/7zip.tar.gz.sha1',
     ],
   },
+  {
+    'name': 'emsdk install',
+    'pattern': '.',
+    'condition': 'download_emsdk',
+    'action': [
+      'python',
+      'src/third_party/emsdk/emsdk.py',
+      'install',
+      'latest',
+    ]
+  },
+  {
+    'name': 'emsdk activate',
+    'pattern': '.',
+    'condition': 'download_emsdk',
+    'action': [
+      'python',
+      'src/third_party/emsdk/emsdk.py',
+      'activate',
+      'latest',
+      '--embedded', # Makes the installation hermetic to //third_party/emsdk
```

Comment:
You don't need `--embedded` anymore, its the default.

Reply:
Do you know what revision?  We have to roll forward to at least that rev.

Reply:
It was revision 858b176 from about 3 months ago.